components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -1346,28 +1346,36 @@ private static void buildOpenIDConnectAuthenticationConfiguration(
         }
 
         property = new Property();
-        property.setName("commonAuthQueryParams");
+        property.setName(IdentityApplicationConstants.Authenticator.OIDC.SCOPES);
+        if (paramMap.get("scopes") != null
+                && paramMap.get("scopes").trim().length() > 0) {
+            property.setValue(paramMap.get("scopes"));
```

Comment:
Don't we have a constant defined for `scopes` ?

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/4765